en_US/faq/use-guide.md

```diff
@@ -52,17 +49,17 @@ The update commands for different installation modes:
 
 ```
 ## zip packages
-./bin/emqx_ctl license reload etc/emqx.lic
+./bin/emqx_ctl license reload path/to/emqx.lic
 
 ## DEB/RPM
-emqx_ctl license reload /etc/emqx/emqx.lic
+emqx_ctl license reload path/to/emqx.lic
 
 ## Docker
-docker exec -it emqx-ee emqx_ctl license reload /opt/emqx/etc/emqx.lic
+docker exec -it emqx-ee emqx_ctl license reload path/to/emqx.lic
 ```
 
 ::: tip
```

Comment:
we should add here that before e4.3.X (I'm not sure which version), `path/to/emqx.lic` should be `etc/emqx.lic`

Reply:
also the fact that in older version, reload only affects the local node.